test/iterators.jl

```diff
@@ -3,6 +3,9 @@
 using Base.Iterators
 using Random
 
+import Base: IteratorSize, SizeUnknown
+@test IteratorSize(Any) isa SizeUnknown
```

Comment:
```suggestion
@test Base.IteratorSize(Any) isa Base.SizeUnknown
```

seems like it would make the test more "local".